lib/rgb_matrix/animation/cycle_all.ex

```diff
@@ -19,6 +19,7 @@ defmodule Xebow.Animation.CycleAll do
     hue = mod(time, 360)
     color = HSV.new(hue, 100, 100)
 
+    # FIXME: no reaching into Xebow namespace
```

Comment:
This would be good to log as tech debt when we merge this. It will probably get lost if it's only a comment.

Reply:
oh no, don't worry, I'm not gonna lose this, it's very top of my mind! :sweat_smile: